Razorpay.php

```diff
@@ -1,9 +1,11 @@
 <?php
 
+//require('src/Request.php');
+
 // Include Requests only if not already defined
 if (class_exists('Requests') === false)
 {
-    require_once __DIR__.'/libs/Requests-1.8.0/library/Requests.php';
+    require_once __DIR__.'/vendor/rmccue/requests/library/Requests.php';
```

Comment:
Please remove require_once __DIR__.'/vendor/rmccue/requests/library/Requests.php'; 
and use libs folder to test

Reply:
M here